packages/jetbrains-plugin/src/main/kotlin/com/mongodb/jbplugin/editor/CachedQueryService.kt

```diff
@@ -99,12 +99,18 @@ class CachedQueryService(
                 }
             }
 
-            CachedValueProvider.Result.create(parsedAst, attachment, connectionStateViewModel)
+            Result.create(parsedAst, attachment, connectionStateViewModel)
         }
 
         attachment.putUserData(queryCacheKey, cachedValue)
-        return decorateWithMetadata(dataSource, attachment.getUserData(queryCacheKey)!!.value)
-    }
+        return@runCatching decorateWithMetadata(dataSource, attachment.getUserData(queryCacheKey)!!.value)
+    }.onFailure { ex ->
+        logger.info(
```

Comment:
```suggestion
        logger.warn(
```
Perhaps warn would be better here?